GPU/GLES/Framebuffer.cpp

```diff
@@ -1656,13 +1656,7 @@ void FramebufferManager::PackFramebufferSync_(VirtualFramebuffer *vfb, int x, in
 void FramebufferManager::EndFrame() {
 	if (resized_) {
 		DestroyAllFBOs();
-		glstate.viewport.set(0, 0, PSP_CoreParameter().pixelWidth, PSP_CoreParameter().pixelHeight);
-		int zoom = g_Config.iInternalResolution;
-		if (zoom == 0) // auto mode
-			zoom = (PSP_CoreParameter().pixelWidth + 479) / 480;
```

Comment:
Ugh, yeah, you are right, we need it here for them. But on Windows it's duplication. I'll ifdef it.